newIDE/app/src/EventsSheet/EventsTree/index.js

```diff
@@ -333,6 +333,7 @@ type EventsTreeProps = {|
   onEventMoved: (previousRowIndex: number, nextRowIndex: number) => void,
   onEndEditingEvent: (event: gdBaseEvent) => void,
   onScroll?: () => void,
+  scrollPosition?: number,
```

Comment:
I would call this initialScrollPosition because changing this props won't impact a component already mounted